torch/fx/experimental/symbolic_shapes.py

```diff
@@ -7325,6 +7325,23 @@ def _print_Symbol(self, sym: sympy.Symbol) -> str:
         return self.src_map[sym.name][0]
 
 
+def _is_non_negative_check(cond: sympy.Basic) -> Optional[str]:
+    """
+    Check if a condition (SymPy expression) is checking for non-negative values (>= 0).
+    Returns the variable name if it's a non-negative check (>= 0), None otherwise.
+    """
+    if isinstance(cond, sympy.Rel):
+        if cond.rel_op == ">=":
+            left = cond.lhs
+            right = cond.rhs
+            if right == 0:
+                if isinstance(left, (sympy.Symbol, sympy.Mul, sympy.Add)):
```

Comment:
can you explain why are we checking the left type here? and then returning name for non symbol?
I do not see any test case for the non symbol case. 
is returning str(left) not sufficient here?

Reply:
@laithsakka returning str(left) is sufficient.